graspologic/plot/plot.py

```diff
@@ -262,17 +263,11 @@ def heatmap(
         if len(xticklabels) != X.shape[1]:
             msg = "xticklabels must have same length {}.".format(X.shape[1])
             raise ValueError(msg)
-    elif not isinstance(xticklabels, bool):
-        msg = "xticklabels must be a bool or a list, not {}".format(type(xticklabels))
-        raise TypeError(msg)
 
     if isinstance(yticklabels, list):
         if len(yticklabels) != X.shape[0]:
             msg = "yticklabels must have same length {}.".format(X.shape[0])
             raise ValueError(msg)
-    elif not isinstance(yticklabels, bool):
-        msg = "yticklabels must be a bool or a list, not {}".format(type(yticklabels))
-        raise TypeError(msg)
```

Comment:
I deleted these because, in `sns.heatmap`, `xticklabels` and `yticklabels` accept an `int` as well -- this specifies the number of values in between ticks (so, for instance, `yticklabels=5` means that ever 5 ticks, plot a label), and I think we want that functionality. Seaborn handles the error-checking for this internally, so I figured it was unnecessary here.